config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,13 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (c Config) Validate() (r report.Report) {
+	for _, fs := range c.Storage.Filesystems {
+		fsPath := *fs.Path
+		if len(fsPath) < 4 || (fsPath[:4] != "/etc" && fsPath[:4] != "/var") {
```

Comment:
Hmm, I think this works but what do you think about maybe using [regex](https://pkg.go.dev/regexp)?